### DISCUSSION TITLE
Output volume on client depends on current volume on windows sunshine host.

### DISCUSSION THREAD

**Discussion by BumblingCoder** [LizardByte/community#314] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Sunshine should either have a way of compensating for the volume, recording the pre-fader stream, or an option to maximize the volume while streaming and then put it back to the previous volume setting.

### Expected Behavior

The volume coming from the client is independent of the current volume setting on the windows desktop that I am streaming from.

### Host Operating System

Windows

### Operating System Version

10

### Architecture

64 bit

### Sunshine commit or version

0.20.0

### Package

Windows - installer

### GPU Type

n/a

### GPU Model

any

### GPU Driver/Mesa Version

any

### Config

```Shell
empty
```

### Relevant log output

```shell
no relevant logs
```

**Comment by Lucide** [comment]
I could swear this is how it was until two or three (stable) releases ago..
Suddenly I'm bitten by this issue on windows 10 build 19045.6093

**Reply by lyaaz** [reply]
> I could swear this is how it was until two or three (stable) releases ago.. Suddenly I'm bitten by this issue on windows 10 build 19045.6093

Turn off all settings like "Audio Enhancement" and "Sound Effects" in device settings would be helpful

**Reply by Lucide** [reply]
Yes, I have audio enhancements turned off on the device and no custom APOs.

I'm using the simple hdmi audio sink of a nvidia gpu. Can you reproduce?

**Comment by Lucide** [comment]
Just to know, does anybody else have this issue, or is it something on my end?
It breaks screensharing because the host would be forced to max his own volume.

I'm sure it used to ignore the volume, and I would often put it very low (or 0) without consequences.
